frontend/javascripts/oxalis/model/reducers/connectome_reducer.js

```diff
@@ -0,0 +1,193 @@
+// @flow
+
+import Maybe from "data.maybe";
+import _ from "lodash";
+import update from "immutability-helper";
+
+import type { Action } from "oxalis/model/actions/actions";
+import type { OxalisState, SkeletonTracing, TreeMap } from "oxalis/store";
+import {
+  addTreesAndGroups,
+  getMaximumNodeId,
+} from "oxalis/model/reducers/skeletontracing_reducer_helpers";
+import { updateKey3 } from "oxalis/model/helpers/deep_update";
+import Constants from "oxalis/constants";
+
+function getSkeletonTracingForConnectome(
+  state: OxalisState,
+  layerName: string,
+): Maybe<SkeletonTracing> {
```

Comment:
Since flow handles nullable types quite well, I'm not too convinced of the value of `Maybe`, anymore. This is why I haven't really adopted the class for new stuff. If you think, it's useful, feel free to keep it, but if you agree with my take, then I'd suggest to not add further usages of it (and in the long run, we could try to remove it along the way).

/edit: Ah, you probably did this to maximize the compatibility with the existing skeleton reducer (helpers). Then, it probably makes sense to keep it as is (at least for the scope of this PR).

Reply:
That makes sense :+1: As you say, in this case it was done to maintain compatibility :)